sql/core/src/test/resources/sql-tests/results/cast.sql.out

```diff
@@ -271,3 +271,19 @@ Extended Usage:
 
 Function: boolean
 Usage: boolean(expr) - Casts the value `expr` to the target data type `boolean`.
+
+
+-- !query 33
+SELECT CAST('interval 3 month 1 hour' AS interval)
+-- !query 33 schema
+struct<CAST(interval 3 month 1 hour AS INTERVAL):interval>
+-- !query 33 output
+interval 3 months 1 hours
+
+
+-- !query 34
+SELECT CAST(interval 3 month 1 hour AS string)
+-- !query 34 schema
+struct<CAST(interval 3 months 1 hours AS STRING):string>
```

Comment:
Ur, this is irrelevant to this PR.
Is this column name a bug? `1 hours` instead of `1 hour`?